ckb-debugger-tests/src/bin/gr-child-script.rs

```diff
@@ -102,29 +90,42 @@ pub fn main() -> Result<(), Box<dyn std::error::Error>> {
             .unwrap()
             .output
             .lock
-            .args = JsonBytes::from_vec(args);
+            .args = JsonBytes::from_vec(lock_wrapper_args);
     }
 
     // next hash is set to 0xFF..FF, maximum one
-    let mut config_cell_data = vec![0xFF; 32];
-    // config cell data filled
-    config_cell_data.extend(child_script_config.as_slice());
-    repr_tx.mock_info.cell_deps.last_mut().unwrap().data = JsonBytes::from_vec(config_cell_data);
+    let mut input_data = vec![0xFF; 32];
+    if clap_args.has_config_cell {
+        let config_cell_data = ConfigCellData::new_builder()
+            .wrapped_script(combine_lock_script.clone())
+            .script_config(child_script_config.as_bytes().pack())
+            .build();
+        input_data.extend(config_cell_data.as_slice());
+    }
+    repr_tx.mock_info.cell_deps.last_mut().unwrap().data = JsonBytes::from_vec(input_data);
 
     let inner_witness = BytesVec::new_builder().push(vec![0u8; 65].pack()).build();
     let config: ChildScriptConfigOpt = if clap_args.has_config_cell {
         None.pack()
     } else {
         Some(child_script_config).pack()
     };
+    let wrapped_script = if clap_args.has_config_cell {
+        None.pack()
+    } else {
+        Some(combine_lock_script).pack()
+    };
     let combine_lock_witness = CombineLockWitness::new_builder()
         .index(Uint16::new_unchecked(0u16.to_le_bytes().to_vec().into()))
         .inner_witness(inner_witness)
         .script_config(config.clone())
         .build();
-
+    let lock_wrapper_witness = LockWrapperWitness::new_builder()
```

Comment:
simplify it with "create_lock_wrapper_witness".

Reply:
The `wrapped_script` and `config` is an optional value, which does not satisfy the definition of the function.